incubator/monochart/values.yaml

```diff
@@ -0,0 +1,108 @@
+replicaCount: 1
+image:
+  repository: nginx
+  tag: 0.1.0
```

Comment:
Let's leave this blank and error if not set.

Reply:
@osterman i'll should revert these changes, because it breaks CI.